LISA/LISAImplementation.cpp

```diff
@@ -241,14 +241,56 @@ class LISAImplementation : public Exchange::ILISA {
 
     virtual uint32_t Register(ILISA::INotification* notification) override
     {
+        // Make sure a callback is not registered multiple times.
+        ASSERT(std::find(_notificationCallbacks.begin(), _notificationCallbacks.end(), notification) == _notificationCallbacks.end());
+
+        _notificationCallbacks.push_back(notification);
+        notification->AddRef();
+
+        TRACE_GLOBAL(Trace::Information, (_T("Register INotification:%p"), notification));
```

Comment:
I used INFO macro here (as well as in other places in code eg. LISAJsonRpc.cpp) but had to revert to old logging way since there is currently a problem with Debug.h - LISA namespace it defines clashes with LISA class. 
"Debug.h:35:11: error: 'namespace WPEFramework::Plugin::LISA { }' redeclared as different kind of symbol
|  namespace LISA {"

I didn't want to fix that in scope of this PR. I guess it would be best to address that issue in a separate PR, fix it and change log entries in the entire code to use the new logger macros.